contrib/terraform/equinix/kubespray.tf

```diff
@@ -10,7 +10,7 @@ resource "equinix_metal_device" "k8s_master" {
   count            = var.number_of_k8s_masters
   hostname         = "${var.cluster_name}-k8s-master-${count.index + 1}"
   plan             = var.plan_k8s_masters
-  facilities       = [var.facility]
+  metro       = [var.metro]
```

Comment:
Could you please check formatting ? `tf fmt` at least, thank you

Reply:
updated